audit/projects/k8s-artifacts-prod/services/compute/project-info.json

```diff
@@ -1,181 +0,0 @@
-{
```

Comment:
This seems wrong - why is it deleted?

Reply:
Unfortunately I don't know. I'm gonna try to run the script once again, but I didn't remote any file manually.

Reply:
Updated